src/inspector/models/notation/frames/textframesettingsmodel.cpp

```diff
@@ -69,6 +69,11 @@ void TextFrameSettingsModel::resetProperties()
     m_frameBottomMargin->resetToDefault();
 }
 
+void TextFrameSettingsModel::updatePropertiesOnNotationChanged()
+{
+    loadProperties();
+}
```

Comment:
couldn't actually see how you could use the mouse/keyboard to modify text frame properties on the score but I'm assuming it's possible